coverage/env.py

```diff
@@ -36,3 +36,11 @@
 # Even when running tests, you can use COVERAGE_TESTING=0 to disable the
 # test-specific behavior like contracts.
 TESTING = os.getenv('COVERAGE_TESTING', '') == 'True'
+
+try:
+    import toml
+except ImportError:
+    TOML_SUPPORT = False
+else:
+    del toml
+    TOML_SUPPORT = True
```

Comment:
I think this clause would be better in backward.py.  I'd prefer something like:
```
try:
    import toml
except ImportError:
    toml = None
```
and then later using "if toml:" to check for toml support.